src/k8s/cmd/k8s/k8s_get_join_token.go

```diff
@@ -60,7 +60,7 @@ func newGetJoinTokenCmd() *cobra.Command {
 		},
 	}
 
-	getJoinTokenCmd.PersistentFlags().StringVarP(&getJoinTokenCmdOpts.outputFormat, "output-format", "o", "plain", "Specify in which format the output should be printed. One of plain, json or yaml")
+	getJoinTokenCmd.PersistentFlags().StringVarP(&getJoinTokenCmdOpts.outputFormat, "output-format", "o", "plain", "set the output format to one of plain (default), json or yaml")
```

Comment:
```suggestion
	getJoinTokenCmd.PersistentFlags().StringVarP(&getJoinTokenCmdOpts.outputFormat, "output-format", "o", "plain", "set the output format to one of plain, json or yaml")
```

Cobra will note the default value already:
```
Create a join token for a node to join the cluster

Usage:
  k8s get-join-token <node-name> [flags]

Flags:
  -h, --help                   help for get-join-token
  -o, --output-format string   Specify in which format the output should be printed. One of plain, json or yaml (default "plain")
      --worker                 generate a join token for a worker node
```

Reply:
Can we check that the flags are in the correct order for all commands? Can we make sure the `--help` flag is always the first. (I believe there might be some inconsistencies across commands)